modules/javafx.swing/src/main/java/javafx/embed/swing/InputMethodSupport.java

```diff
@@ -45,6 +36,13 @@
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
+import javafx.collections.ObservableList;
+import javafx.geometry.Point2D;
+import javafx.scene.input.InputMethodHighlight;
+import javafx.scene.input.InputMethodTextRun;
+import com.sun.javafx.application.PlatformImpl;
+import com.sun.javafx.collections.ObservableListWrapper;
+import com.sun.javafx.scene.input.ExtendedInputMethodRequests;
```

Comment:
Probably better to do this one separately. It seems OK to combine with another module, but I'd prefer it separate from what is otherwise a test-only change.

Reply:
> Probably better to do this one separately.

will do separately.  

can I at least combine swing and web modules (there is only one file in swing that needs an update)?